test_torch_tensor_dag.py

```diff
@@ -0,0 +1,135 @@
+# coding: utf-8
+import logging
+import os
+import sys
+import torch
+import pickle
+import io
+
+import pytest
+
+import ray
+from ray.air._internal import torch_utils
+import ray.cluster_utils
+from ray.dag import InputNode
+from ray.tests.conftest import *  # noqa
+
+from ray.experimental import TorchTensor
+
+
+logger = logging.getLogger(__name__)
+
+
+
+@ray.remote
+class TorchTensorWorker:
+    def __init__(self):
+        self.device = torch_utils.get_devices()[0]
+
+    def send(self, shape, dtype, value: int):
+        return torch.ones(shape, dtype=dtype, device=self.device) * value
+
+    def recv(self, tensor):
+        assert tensor.device == self.device
```

Comment:
How would the multi-device case be handled in the DAG declaration? I imagine it would have to be something you also include as part of the DAG in order to set up the right deserialization processing.

Reply:
Yeah, I was thinking to expose a call on the actor that you can use to set the default device.